### ISSUE TITLE
Get-Boxstarter Parsing -version resulted in exception: Cannot bundle unregistered option '-e'

### ISSUE THREAD

**Issue by berney** [chocolatey/boxstarter#206]
During Get-Boxstarter installing Boxstarter there's an error message: 

```
Welcome to the Boxstarter Module installer!
Chocolatey installed, Installing Boxstarter Modules.
Parsing -version resulted in exception:
 Cannot bundle unregistred option: '-e'.
Chocolatey v0.10.0
```

It seems that `--version` should be used instead of `-version`. I'm not sure if Chocolatey is going to do anything to improve this as they have left this bug https://github.com/chocolatey/choco/issues/842 open.

If I get the time I'll dig into this further and send a Pull Request.

**Comment by pauby** [comment]
As the parameter is `--version` (ie. a double dash) then this is expected behaviour is you use a single dash. 

Closing this issue but we can reopen it again in the future if needed.

**Comment by berney** [comment]
This was a problem in function Get-Boxstarter, where the code was trying to use `-version`, which is wrong (missing a dash). It was fixed in da818feddd573267b1e10c490628e653800d1637
